RyleyGG/apluschiever
# Lesson page

# Issue
#33 
closes #33

# Description
Developed lesson page. Currently it is set up with hard coded lesson content, but uploading lesson content should be pretty simple.

# Type Of Change

# Test Steps
Does not break anything. Tested by trying out different set-ups, moving content around.
Only issues that have not been properly accounted for are faulty URLs (both invalid urls and urls that may not be displayed). However, I think the issue of urls that cannot be displayed could be set-aside for the moment and invalid urls should be accounted for when uploading content.

# Checklist

## Formatting
Please use markdown in your pull request message. A useful summary of commands can be found [here](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf).
